src/Components/Patient/PatientRegister.tsx

```diff
@@ -701,6 +728,7 @@ export const PatientRegister = (props: PatientRegisterProps) => {
       emergency_phone_number: parsePhoneNumber(formData.emergency_phone_number),
       date_of_birth: dateQueryString(formData.date_of_birth),
```

Comment:
backend throws format validation error if no date_of_birth is present as `dateQueryString(null)` would be `""`.

```suggestion
      date_of_birth: dateQueryString(formData.date_of_birth) || null,
```